### DISCUSSION TITLE
Lost XCH

### DISCUSSION THREAD

**Discussion by Kerdarn** [Chia-Network/chia-blockchain#14770] [Support]
Hello,
From my wallet was stollen XCH, but i dont know why there are no transaction on chia client software, my wallet for scan is xch16pccjr4eenedaxa9s9kd5kyvsc5x97kdc40quh8sdezez7qwyjpq949zr2 and on chia scan there is trx id  0x1f2e8f8852c8f12797514601f3a9ad91a8ecbd2a88adeb03bde94e0d3be81716 that couln`t find on chia client. how is that possibile and how hacker could stollen xch. My mnemonic was written only on paper and noone have access to my PC
![chia](https://user-images.githubusercontent.com/127246237/223529349-fa7d3110-20fd-486f-acd5-901f3a098324.jpg)

**Comment by shaneo257** [comment]
do you use any 3rd party farm monitoring tools on the PC?

**Reply by Kerdarn** [reply]
No, only chia official client v.1.7.0 and in the time of transaction I was in front of PC, (watching netflix ) so the transaction wasn't done from my pc. How i write above, the most strange thing is that on chia client I couldn't find transaction, but found them on chia scan.
